projects/go-lib/src/lib/go-config.model.ts

```diff
@@ -1,5 +1,78 @@
+/**
+ * Theme colors for Goponents
+ */
+export enum ThemeColors {
+  brand = '#65B360',
+  dark = '#202626',
+  light = '#ffffff'
+}
+
+
+/**
+ * What mode of branding to enable
+ */
+export enum BrandingMode {
+  /**
+   * Company mode is for branding the app in accordance
+   * to the constaints of the base company (ex. Tangoe)
+   */
+  company = 'Company',
+  /**
+   * Client mode is for branding the app in accordance
+   * to the standards for a client company (ex. External Branding / White-labelling)
+   */
+  client = 'Client'
+}
```

Comment:
What is the actual difference between a company branded app and a client branded app here? The only thing that I see that uses this is the background of the header currently and even there it would seem that it's the difference between setting it to the company color, or setting it to white. I guess my question here is does Client Mode brand the app with External Branding as stated here, or is it just setting everything to the light theme?

Reply:
@StevenUlmer so there's an important distinction here between branding and theme. A theme would be light vs. dark. Branding is the color applied as an accent. In this particular case, the only main thing that this changes is how the branding color get's applied to the background of the header.

Reply:
Ya I guess I was wondering if there might be a more descriptive variable name for these two variables based on what they do, as company vs client branding doesn't really tell me the difference, but honestly I can't think of a good name that is more descriptive so this is probably fine

Reply:
Guys, this is super trivial, but check line #17, there is a typo: ~constaints~ constraints.
@StevenUlmer / @grahamhency